project_planning/reset_planning_wizard_view.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_wizard_reset_planning" model="ir.ui.view">
```

Comment:
Is there a ACL to launch the wizard?
We should have 2 levels: Manager and user
* Planning manager can reset for all users.
* Planning users can only reset for themselves